qi-lib/flow/extended/expander.rkt

```diff
@@ -72,8 +72,7 @@
     #:description "a flow expression"
     #:binding-space qi
 
-    (gen e:expr ...)
-    #:binding (host e)
+    (gen e:racket-expr ...)
```

Comment:
Any reason `racket-expr` rather than `host-expr`?

Reply:
Consistency with the `racket-var` and `racket-macro` features. And these all really only do work with Racket as the "host" so far. Perhaps a version 2 will eventually support towers of languages properly.